pkg/health/heartbeat/heartbeat.go

```diff
@@ -129,6 +132,7 @@ func (hb *Heartbeat) RenewMemberLease(ctx context.Context) error {
 	renewedMemberLease.Spec.HolderIdentity = &memberID
 	renewedTime := time.Now()
 	renewedMemberLease.Spec.RenewTime = &metav1.MicroTime{Time: renewedTime}
+	renewedMemberLease.Annotations = hb.metadata
```

Comment:
Please only overwrite the `PeerUrlTLSEnabledKey`, not the entire annotations.

Reply:
Fixed now